File: src/main/java/com/addressbook/AddressBookApplication.java
package com.addressbook;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SuppressWarnings("unused")
@SpringBootApplication(scanBasePackages = "com.addressbook")
public class AddressBookApplication {
    public static void main(String[] args) {
        SpringApplication.run(AddressBookApplication.class, args);
    }
}

File: addressbook-main/src/main/kotlin/com/addressbook/AddressBookApplication.kt
package com.addressbook

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.netflix.eureka.EnableEurekaClient
import org.springframework.cloud.openfeign.EnableFeignClients

@EnableEurekaClient
@EnableFeignClients
@SpringBootApplication(scanBasePackages = ["com.addressbook"])
open class AddressBookApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(AddressBookApplication::class.java, *args);
        }
    }
}